### ISSUE TITLE
bug: removed models showing as a price change to "free"

### ISSUE THREAD

**Issue by sambarnes** [fry69/orw#1]
hey! found this repo through the OR discord, looks slick! love the native RSS support

@spaceemotion brought up a great point about our main site lacking a good changelog for bulk model addition/changes/deletions, and then i remembered seeing your site posted

when we were looking through some of [the recent change entries](https://orw.karleo.net/changes), we noticed that it detected our removal of airoboros as a price change:
![image](https://github.com/fry69/orw/assets/1211977/473b6876-2132-49cb-b7a8-1572fff85412)

after checking our `/v1/models` api result, i think i see the issue:
![image](https://github.com/fry69/orw/assets/1211977/c1434c1c-3212-4fa3-bfa7-cd63ae988eda)

confusion probs comes from the fact that when we "remove" a model, we try to keep the model's config visible via API & frontend (for analytics purposes, SEO, etc).  usually avoid yoinking the model from the API completely

so when a model is removed from the last provider, it's price will be set to `-1`

---

happy to try and knock out a quick pr for this here if you think its appropriate! but also feel free to take it on whenever if you would rather that

cheers!

**Comment by spaceemotion** [comment]
Doesn't the `auto` and `model of the week` also use `-1` as pricing? might make sense to use a different notation then? 🤔

**Comment by sambarnes** [comment]
> Doesn't the `auto` and `model of the week` also use `-1` as pricing? might make sense to use a different notation then? 🤔 

good point, yeah we're in the process of starting to formalize the models API a bit more in the next few days.  will def be iterating on the schema there
